src/pages/server-sitemap.xml/index.tsx

```diff
@@ -1,20 +1,20 @@
 import { getServerSideSitemap } from 'next-sitemap'
 import { GetServerSideProps, GetServerSidePropsContext } from 'next'
+import { allDocsUrls, allTutorialsUrls } from 'lib/sitemap'
 
 export const getServerSideProps: GetServerSideProps = async (
 	ctx: GetServerSidePropsContext
 ) => {
-	const response = await fetch(
-		`https://content.hashicorp.com/api/all-docs-paths`
-	)
+	// returns an array of docs content sitemap objects per slug
+	const docsUrlsPromise = allDocsUrls()
+	// returns an array of tutorials content sitemap objects per slug
+	const tutorialsUrlsPromise = allTutorialsUrls()
+	const [docsUrls, tutorialsUrls] = await Promise.all([
+		docsUrlsPromise,
+		tutorialsUrlsPromise,
+	])
```

Comment:
Might want to consider some error handling here?

Reply:
Added!